contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -1749,8 +1750,24 @@ pltsql_sequence_datatype_map(ParseState *pstate,
 		return;
 
 	type_def = defGetTypeName(as_type);
+	type_names = type_def->names;
+	switch (list_length(type_def->names))
+	{
+		case 2:
+			strVal(linitial(type_names)) = get_physical_schema_name(get_cur_db_name(),strVal(linitial(type_names)));
+			break;
+		case 3:
+			strVal(lsecond(type_names)) = get_physical_schema_name(strVal(linitial(type_names)),strVal(lsecond(type_names)));
```

Comment:
This case is for db.schema.name, right?
In that case List will look like db,schema,name (list length of 3)
We need to convert it to db_schema,name (list length of 2)

what you have done makes the list look like: db,db_schema,name (list length of 3)